src/IO/WriteHelpers.h

```diff
@@ -798,6 +798,28 @@ inline void writeDateTimeTextRFC1123(time_t datetime, WriteBuffer & buf, const D
     buf.write(" GMT", 4);
 }
 
+inline void writeText(const String & x, WriteBuffer & buf) { writeEscapedString(x, buf); }
+
+inline void writeDateTimeTextUnixTimestamp(time_t datetime, WriteBuffer & buf, const DateLUTImpl & date_lut = DateLUT::instance())
```

Comment:
You don't need this function.

unix timestamp is the `time_t` itself.
It is independent of time zone.

Reply:
Fixed. Thanks.